.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
 <!-- Feel free to remove check-list items aren't relevant to your change -->
 
 - [ ] Closes #xxxx
-- [ ] Tests added
+- [ ] Tests added, and functions annotated with `-> None:`
```

Comment:
Let's discuss this more back in #5690, but I don't think I would understand what this means from the description here alone.